merlin/systems/dag/runtimes/triton/runtime.py

```diff
@@ -220,11 +213,11 @@ def _executor_model_export(
         return config
 
 
-def _create_node_table(nodes, backend):
+def _create_node_table(nodes):
     exportable_node_idx = 0
     node_id_lookup = {}
     for node in nodes:
-        if node.exportable(backend):
+        if isinstance(node.op, TritonOperator):
```

Comment:
We don't need to check the `exportable` method anymore because every `TrtionOperator` is expected to implement `export` to save the operator as a Triton model.